src/Core/Serialization/ProjectVersionMigrator.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Reko.Core.Serialization
+{
+    public static class ProjectVersionMigrator
+    {
+        public static Project_v5 MigrateProject(Project_v4 v4) => new Project_v5()
+        {
+            ArchitectureName = v4.ArchitectureName,
+            PlatformName = v4.PlatformName,
+            InputFiles = v4.InputFiles.Select(i => MigrateDecompilerInput(i)).ToList(),
+            MetadataFiles = v4.MetadataFiles,
+            AssemblerFiles = v4.AssemblerFiles,
+        };
+
+        /// <summary>
+        /// Ignored fields (They existed in v4 but are unused in current code):
```

Comment:
<Summary> explains what teh method is for. Put this in a <remarks> element further down.

Reply:
That's correct, but I'm not sure if my new comments are of an appropriate scope either. Feedback is much appreciated for improvements.